Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml

```diff
@@ -58,6 +58,8 @@
     ClothingHeadFishCap: 2
     ClothingHeadRastaHat: 2
     ClothingBeltStorageWaistbag: 3
+    ClothingHeadHatGladiator: 1
+    ClothingUniformJumpsuitGladiator: 1
```

Comment:
Please don't put more clothes in clothesmate as it's already bloated.

Reply:
wait, i through i put it in theater drobe...